app_python/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.9"
 services:
   web:
-    image: bragov4ik/time_web_app:latest
```

Comment:
Nit: you can include both `image` and `build`. using that, docker-compose will build only on `--build` flag, and pull image otherwise